modules/twinkleblock.js

```diff
@@ -335,6 +340,20 @@ Twinkle.block.callback.change_action = function twinkleblockCallbackChangeAction
 		}
 	}
 
+	// DS selection visible in either the template field set or preset,
+	// joint settings saved here
+	var dsSelectSettings = {
+		type: 'select',
+		name: 'dstopic',
+		display: 'none',
```

Comment:
```suggestion
		style: 'display: none',
```
right?

Reply:
Hmm, yes.  `block` has quite a few of these, stemming back to the beginning.

Reply:
Did away with 'em all